apps/super-app/src/index.tsx

```diff
@@ -3,14 +3,15 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
+  <Router>
```

Comment:
This is my personal taste, but let's move this into the `App` component instead of outside of it. I would prefer to be able to see all of the providers at once instead of having some providers inside `App` and some outside of it.